libs/wingc/src/jsify.rs

```diff
@@ -1458,7 +1458,7 @@ impl<'a> JSifier<'a> {
 
 			// TODO Hack to make sure closures match the IInflight contract from wingsdk
 			if class_type.is_closure() {
-				code.line("_hash = require('crypto').createHash('md5').update(this._toInflight()).digest('hex');")
+				code.line("_hash = $helpers.closureId();")
```

Comment:
Not sure if $helpers is the right place since it's only needed in preflight. I'll move it to stdlib.core